e2e/test-data/smoke-test-data/smoke_pool_sc_pvc.tmpl

```diff
@@ -0,0 +1,43 @@
+apiVersion: rook.io/v1alpha1
+kind: Rookpool
+metadata:
+  name: replicapool
+  namespace: rook
+spec:
+  replication:
+    size: 1
+  # For an erasure-coded pool, comment out the replication count above and uncomment the following settings.
+  # Make sure you have enough OSDs to support the replica count or erasure code chunks.
+  #erasureCode:
+  #  codingChunks: 2
+  #  dataChunks: 2
+---
+apiVersion: storage.k8s.io/v1beta1
+kind: StorageClass
+metadata:
+   name: rook-block
+   namespace: rook
+provisioner: kubernetes.io/rbd
```

Comment:
these look out of date and should be using the new operator's provisioner.  i've been doing that for some of them so maybe we can sync up on what those look like.